### ISSUE TITLE
Text of buttons and links should not be repeated in the image alternative

### ISSUE THREAD

**Issue by pdehaan** [mozilla/send#438]
via #139 

```
line 21   col 7   Error  Text of buttons and links should not be repeated in the image alternative                                                                                 axe
```

Not entirely sure I know what duplicate `alt=""` this is talking about, but we're doing a bit of magic with l20n. Even w/ cURL and DOM inspectors, I can't seem to find a line 21 that matches anything remotely close to an `alt=""` or `title=""` attribute.

**Comment by varghesethomase** [comment]
The bug here is the alt attribute for image has text "Send" and the text wrapped by the h1 tag is again "Send". <img src="/resources/send_logo.svg" alt="Send"/><h1 class="site-title">Send</h1>

**Comment by varghesethomase** [comment]
If you can get me the verbiages to be used, I will take this up.

**Comment by dannycoates** [comment]
fixed in #474
